distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -2447,6 +2453,82 @@ private void checkExpectations(CometDClient cometDClient, Integer numNotificatio
 
     }
 
+    /**
+     * Confirms that the "download resource to local cache" action's url can be
+     * found in the CometDClient messages, and compared to the expected url.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void findMetacardActionUrlUsingCometDClient() throws Exception {
+
+       String metacardId = ingestXmlWithProduct(testName.getMethodName() + ".txt");
+
+        metacardsToDelete.add(metacardId);
+
+        final String RESPONSE_CHANNEL = "/0193d9e7f9ed4f8f8bd02103143c41d6";
```

Comment:
Format this. It appears that you have inconsistent indenting here. It might be the display on github, but it looks like it's really off.